README.md

```diff
@@ -1,3 +1,18 @@
+## Maintenance Countdown:
+Commencing from January 1, 2024, a 90-day countdown has been initiated, signaling the transition of Web3.js version 1.x into an end-of-maintenance phase.
+
+### Timeline of Changes:
+
+- 90-Day Countdown (1/1/24 - 3/31/24):
+  During this period, we strongly encourage users to plan accordingly and initiate the upgrade to Web3.js version 4.x
```

Comment:
lgtm, I'll suggest to add migration guide link as well

Reply:
links were added
<img width="823" alt="image" src="https://github.com/web3/web3.js/assets/9366141/7aebb863-d765-42ca-b2a0-2ba19298b88c">
<img width="841" alt="image" src="https://github.com/web3/web3.js/assets/9366141/305361ae-8432-4405-9b4a-f585f5d38a46">